### PR TITLE
codelive/trygger

### PR DESCRIPTION
**Pull request by codefriar**
### What does this PR do?

Adds Metadata driven triggers and associated test recipes.

### CODE REVIEW

File: force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml
Comment (msrivastav13) [inline force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml:3]: I think we should bump this to `51.0`

Comment (codefriar) [reply force-app/main/default/classes/MDTSecondAccountTriggerHandler.cls-meta.xml:3]: good catch. fixing.

**Comment by codefriar** [comment]
> good catch. fixing. 

Fixed.

File: force-app/main/default/classes/Trigger Recipes/sampleHandler.cls
Comment (msrivastav13) [inline force-app/main/default/classes/Trigger Recipes/sampleHandler.cls:12]: Maybe for consistency, we change it to `SampleHandler `!

File: package.json
Comment (msrivastav13) [inline package.json:44]: @codefriar was this intentional? I defer this to @pozil as I am not sure if this is needed!

**Review by msrivastav13 — COMMENTED** [review]
Looks good to me except for Salesforce CLI npm dependency!

I will defer to @pozil on that!

File: package.json
Comment (pozil) [reply package.json]: Thanks for removing it @codefriar. Is the `package-lock.json` up to date with that change?

**Review by pozil — APPROVED** [review]
Looking good. Can you run prettier and address the open items before merging?

**Pull request merged**
